### DISCUSSION TITLE
Actividad 1 - GRUPO 8

### DISCUSSION THREAD

**Discussion by tomaszbk** [frlp-utn-ingsoft/catedra#88] [Show and tell]
https://github.com/Salani1003/vetsoft

**Comment by RodrigoJacznik** [comment]
Buenas!
La actividad está bien resuelta, queda aprobada.
Les dejamos algunos comentarios para que tengan en cuenta en la próxima:

- Se puede mejorar la estandarización de los mensajes de commit. Algunos comienzan con la acción ("fix", "docs") y otros no.
- Si bien no es algo generalizado, hay algunos mensajes de commits muy genéricos por ejemplo: "forms add" o "model added". Estos mensajes capaz tienen sentido en el contexto de un branch pero sin ese contexto no.

Buen trabajo!
Saludos
